### DISCUSSION TITLE
Problem with AUTO_FLOW_REDUCE in the WEL package

### DISCUSSION THREAD

**Discussion by Amir1408** [MODFLOW-ORG/modflow6#1642] [General]
Hi, 

I set a value for AUTO_FLOW_REDUCE, yet pumping rates remain unaffected, causing the head to drop well below the bottom elevation of the layer. I checked the observation file and pumping rate did not change. I am using mf6.4.2. Could there be a bug in WEL package?

# MODFLOW6 Well Boundary Package
BEGIN OPTIONS
  AUXILIARY  CONC1
  SAVE_FLOWS
  PRINT_INPUT
  PRINT_FLOWS  
  AUTO_FLOW_REDUCE 0.1
  AUTO_FLOW_REDUCE_CSV FILEOUT well_reduce.csv
  OBS6  FILEIN  PRW_WEL.obs
END OPTIONS

**Comment by christianlangevin** [comment]
The auto_flow_reduce option only works if the cell is marked as unconfined (ICELLTYPE > 0) in the NPF Package.  Does this explain your situation?  We will clarify this in the user guide.

**Reply by Amir1408** [reply]
Thanks. I changed the ICELLTYPE and the auto_flow_reduce option worked.
